docs/dictionary/property/dontUseQT.lcdoc

```diff
@@ -16,24 +18,38 @@ Example:
 set the dontUseQT to true
 
 Example:
-if the platform is "Win32" then set the dontUseQT to true
+set the dontUseQT of player "video" to false
 
 Value (bool):The <dontUseQT> is true or false.
-By default, the <dontUseQT> <property> is set to false.
+On Windows, the default value of <dontUseQTEffects> is false. On OSX, the 
+default value of the <dontUseQTEffects> is true unless the 32 bit
+version is running on a system under 10.8. 
 
 Description:
-Use the <dontUseQT> <property> to test operation of a <stack> using the built-in MCI video on <Windows|Windows systems>, or to use built-in visual effects rather than <QuickTime> effects.
-
-If you have already used QuickTime during the current session, setting the <dontUseQT> <property> to true has no effect, because the code for LiveCode to use <QuickTime> is already <loaded into memory> and will continue to be used until you quit the application.
-
-The following actions cause QuickTime to be loaded into memory:
-        * Displaying a player object
-        * Playing an audio clip or video clip
-        * Recording sound using the record sound command
-        * Using the recordFormats <function> or the <answer record> command
-        * Using a visual effect with visual effect, <show>, <hide>, or unlock screen
-        * Using the answer effect <command> or <QTEffects> function
-        * Using the <QTVersion> function
+On Windows, when <dontUseQT> is true players will use the DirectShow API
+for multimedia playback.
+
```

Comment:
The QT player is gone entirely from Windows in 8.1 and the dontUseQT property is ignored for players (both the global and the player-local properties).